src/radical/pilot/agent/scheduler/continuous.py

```diff
@@ -381,15 +381,32 @@ def schedule_unit(self, unit):
         is_last  = False
         tag      = cud.get('tag')
 
+        # - PRRTE related - start -
+        dvm_hosts_list = self._rm_lm_info.get('dvm_hosts') or []
```

Comment:
this one as a small "protection" for an incorrect empty value, in case value for `self._rm_lm_info['dvm_hosts']` could be as `None` or `''`. Kinda I like to be sure that a certain key is in checked dictionary and the corresponding value is a non-empty, otherwise I use default value on my own (empty list in this case)